lib/task_logger.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "json"
+require "time"
+
+class TaskLogger
+  # Initialize logger with default parameters
+  def initialize( task, identifier, type, subtype )
```

Comment:
It looks like in all cases so far you have the `subtype` set to rails. Would you ever want it to be anything else in this codebase? If not, I'd remove the parameter and hard code it. If so, it might be good to have a comment with some possible suggestions

Reply:
subtype is "cloudfront" for a few of them (coming from the production_schedule.rb)

Reply:
OK. We could add a default value here (`subtype = "rails"`) and that way in this codebase we wouldn't have to repeat that parameter when creating instances. Not a big deal - just want to have code be as obvious as possible and if other people are working here wonder why we're always sending "rails" and what other values could be sent